ui/src/components/device/DeviceDetails.vue

```diff
@@ -57,13 +57,13 @@
         <v-spacer />
       
         <TerminalDialog :uid="device.uid" />
-      
-        <v-btn icon>
-          <v-icon @click="remove()">
-            delete
-          </v-icon>
-        </v-btn>
-      </v-toolbar>
+
+        <DeviceDelete   
```

Comment:
Please, you should review your code. In your pull request, you have lines and spaces.
You want to improve the code, you can create another commit.